general_itests/steps/deployments_json_steps.py

```diff
@@ -124,4 +123,11 @@ def step_impl_then(context):
         },
     }
     assert expected_deployments == deployments, "actual: %s\nexpected:%s" % (deployments, expected_deployments)
-    shutil.rmtree(context.test_git_repo_dir)
+
+
+@then(u'that deployments.json has a desired_state of "{state}"')
```

Comment:
can you call this `expected_state` instead of just `state` here?
I think it reads better when you do the assertion